src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -172,7 +236,7 @@ const styles = StyleSheet.create({
     paddingTop: dimen.normalizeDimen(12) + top,
     zIndex: 10
   }),
-  Header: {
+  headerContainer: {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The style definition for the headerContainer seems fine.



<!-- This is an auto-generated comment by OSS CodeRabbit -->